<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QKxDesktopToolForm</class>
 <widget class="QWidget" name="QKxDesktopToolForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QToolButton" name="btnFullScreen">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="feidesk.qrc">
       <normaloff>:/resource/skin/fullscreen.png</normaloff>:/resource/skin/fullscreen.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnExitFull">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="feidesk.qrc">
       <normaloff>:/resource/skin/exitfull.png</normaloff>:/resource/skin/exitfull.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnMinimize">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="feidesk.qrc">
       <normaloff>:/resource/skin/minize2.png</normaloff>:/resource/skin/minize2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnNextRatio">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="feidesk.qrc">
       <normaloff>:/resource/skin/scroll.png</normaloff>:/resource/skin/scroll.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnQuality">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="feidesk.qrc">
       <normaloff>:/resource/skin/signal.png</normaloff>:/resource/skin/signal.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnAudioOn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="feidesk.qrc">
       <normaloff>:/resource/skin/audio_on.png</normaloff>:/resource/skin/audio_on.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnAudioOff">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="feidesk.qrc">
       <normaloff>:/resource/skin/audio_off.png</normaloff>:/resource/skin/audio_off.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnFtp">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="feidesk.qrc">
       <normaloff>:/resource/skin/ftp3.png</normaloff>:/resource/skin/ftp3.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnPrivacyOn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="feidesk.qrc">
       <normaloff>:/resource/skin/privacy_on.png</normaloff>:/resource/skin/privacy_on.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnPrivacyOff">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="feidesk.qrc">
       <normaloff>:/resource/skin/privacy_off.png</normaloff>:/resource/skin/privacy_off.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnLock">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="feidesk.qrc">
       <normaloff>:/resource/skin/lock2.png</normaloff>:/resource/skin/lock2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnBlack">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="feidesk.qrc">
       <normaloff>:/resource/skin/wallpaper.png</normaloff>:/resource/skin/wallpaper.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnNextScreen">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="feidesk.qrc">
       <normaloff>:/resource/skin/screen.png</normaloff>:/resource/skin/screen.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnClose">
     <property name="text">
      <string>btn</string>
     </property>
     <property name="icon">
      <iconset resource="feidesk.qrc">
       <normaloff>:/resource/skin/stop.png</normaloff>:/resource/skin/stop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="feidesk.qrc"/>
 </resources>
 <connections/>
</ui>
